install/kubernetes/cilium/dev-test.yaml

```diff
@@ -0,0 +1,671 @@
+---
```

Comment:
Is this new file intentionally added? How is it updated?

Reply:
I made a mistake in a prior commit and rolled this back

Reply:
The file is still there but this conversation is marked as resolved. Based on your comment, I think it was not intentionally added, right?

Reply:
I missed this file. @seanmwinn please remove it before merging the PR. No need to re-run the CI once the PR is fixed as the tests have passed.